hic_qc.py

```diff
@@ -385,6 +390,16 @@ def update_mapped_pair_stats(self, a, b):
                 self.stats['pairs_greater_10k'] += 1
             if dist == 0:
                 self.stats['zero_dist_pairs'] += 1
+            if dist >= 0 and dist < 1000:
```

Comment:
Scientific notation might be easier to read here: `if dist >= 0 and dist < 1e3:` etc.

Reply:
You could also do `if 0 <= dist < 1000`.

Reply:
I'd like to see these in an `if elif else` block. It would also be good to add a comment explaining the purpose of this code, since it sort of duplicates some existing functionality.

Reply:
This is a great point. This should be in if/elif blocks to reduce the likelihood of a future edit inadvertently changing the logical flow through the code